content/blog/apl/2020-07-11_repository_identification.md

```diff
@@ -0,0 +1,99 @@
++++
+type = "blog"
+title = "Haiku Repository Files and Identifiers"
+author = "apl"
+date = "2020-07-11 11:42:49+12:00"
+tags = ["haiku", "software", "repository"]
++++
+
+Software on a computing platform such as Haiku is typically distributed as a package.  Without a packaging system it would be hard for users to install software and because software often depends on other software, the chain of dependencies would be  difficult for a user to resolve themselves.  To orchestrate the distribution and management of the packages, Haiku has a packaging system which consists of applications, online tools, on-host tools and software libraries.  One aspect of the packaging system is the coordination and identification of repositories.
+
```

Comment:
Do we still do "folds"? If so, this would be the right place to insert a

`< !--more-->`

(Do mind the empty line before and after.)
[No space before the ! ... I had to add that to please github's preview...]

Reply:
Thanks @humdingerb ; I've made those changes now.